fedimint-server/src/config/distributedgen.rs

```diff
@@ -473,3 +476,74 @@ mod tests {
         keys
     }
 }
+
+// TODO: this trait is only needed to break the `DkgHandle` impl
+// from it's defintion that is still in `fedimint-core`
+#[async_trait]
+pub trait DkgHandleOps {
+    async fn run_g1<T>(&self, v: T) -> DkgResult<HashMap<T, DkgKeys<G1Projective>>>
```

Comment:
We might want to keep this trait for mocking in tests, making it a `dyn` interface.